modules/tinymce/src/plugins/insertdatetime/main/ts/api/Commands.ts

```diff
@@ -11,12 +11,12 @@ import * as Actions from '../core/Actions';
 import * as Settings from './Settings';
 
 const register = (editor: Editor): void => {
-  editor.addCommand('mceInsertDate', () => {
-    Actions.insertDateTime(editor, Settings.getDateFormat(editor));
+  editor.addCommand('mceInsertDate', (_ui, value) => {
+    Actions.insertDateTime(editor, value ?? Settings.getDateFormat(editor));
   });
 
-  editor.addCommand('mceInsertTime', () => {
-    Actions.insertDateTime(editor, Settings.getTimeFormat(editor));
+  editor.addCommand('mceInsertTime', (_ui, value) => {
```

Comment:
I missed this before sorry but did we want a value passed here? As the buttons only technically require the above to allow a value to be passed.

For consistency I think it's worthwhile, but just wanted to double check.

Reply:
Happy to do either way, I added it for consistency